### PR TITLE
Few changes to get the project to be fully automatable

### PR DESCRIPTION
**Pull request by techgnosis**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -64,27 +64,51 @@ spec:
...
+
 
   stages {
     stage('Setup access') {
       steps {
         container('k8s-node') {
-          script {
-                env.KEYFILE = "/home/jenkins/dev/jenkins-deploy-dev-infra.json"
-            }
+          // env.CLUSTER_ZONE will need to be updated to match the
```
Comment (erkolson) [inline Jenkinsfile]: Need to add env variables for `USER_PASSWORD` and `PG_ADMIN_CONSOLE_PASSWORD`, otherwise scripts require user input.

Comment (techgnosis) [reply Jenkinsfile]: Yeah these environment variables will be set in the Jenkins job. It's outside the scope of these files.

I added teardown to the post step.

I don't really want to mess with peoples gcloud config unless I have to. If they want to turn that off they can.

Comment (erkolson) [reply Jenkinsfile]: Sounds good

File: Jenkinsfile [hunk 2]
```diff
@@ -64,27 +64,51 @@ spec:
     }
   }
 
+  environment {
+        GOOGLE_APPLICATION_CREDENTIALS    = '/home/jenkins/dev/jenkins-deploy-dev-infra.json'
+    }
```
Comment (robinpercy) [inline Jenkinsfile]: whitespace issue here :)

File: Jenkinsfile (hunk 1 above)
Comment (robinpercy) [reply Jenkinsfile]: @techgnosis can/should we add some validation to check that required env vars are set?  I suspect we get strange behavior if one is missed?

**Pull request merged**